components/post_view/post_list/index.js

```diff
@@ -65,6 +66,7 @@ function makeMapStateToProps() {
                 const latestPostId = memoizedGetLatestPostId(postIds);
                 const latestPost = getPost(state, latestPostId);
                 latestPostTimeStamp = latestPost.create_at;
+                latestAriaLabelFunc = (postId, intl) => createAriaLabelForPostId(state, postId, intl);
```

Comment:
You're going to run into that same issue here, but you might need to do it a bit differently since you don't actually know the post ID at this point. Perhaps you could write some sort of `PostReader` component that's only mounted by the `PostList` when necessary. That component could then construct the function with the correct post ID when necessary.